cppSlint/README.md

```diff
@@ -0,0 +1 @@
+# C++ Slint Template Specific Documentation
```

Comment:
will this be empty? I think there is nothing very different from cConsole or cppConsole, we can use the same text, no?

Reply:
This Slint ones I left empty to ask the Slint people to add it, as they will be able to provide better information

Reply:
ok then